protractor/local.config.ts

```diff
@@ -1,7 +1,13 @@
 import { Config } from 'protractor';
+import { reporter } from './helpers/reporter';
+
 
 export const config: Config = {
   framework: 'jasmine',
   specs: [ '../test/google.spec.js' ],
   seleniumAddress: 'http://localhost:4444/wd/hub'
+  onPrepare: () => {
```

Comment:
this is missing a comma `,` at the end of line 8, please make sure the report looks like point 3 of the workshop and add a new screenshot

https://github.com/testing-community/workshop-protractor#3-agregando-reporte-a-la-consola